spin.toml

```diff
@@ -262,6 +282,26 @@ environment = { DESTINATION = "https://developer.fermyon.com/spin/kv-store-tutor
 source = "modules/redirect.wasm"
 environment = { DESTINATION = "https://developer.fermyon.com/spin/cli-reference" }
 
+# Redirect /spin/kubernetes-dynamic-configuration to /spin/spinkube
+[component.redirect-spin-kubernetes-dynamic-configuration]
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "/spin/spinkube" } 
```

Comment:
Have we checked that redirect-to-path works?  I still kind of expect this would need a full URL like all the other redirects have.

Reply:
Yes, I have tested this on localhost